docs/src/ref_impl/build_and_run.md

```diff
@@ -148,7 +149,7 @@ Before you begin, check device-independent [prerequisites](./build_and_run.md#pr
 In the case of i.MX8, Ghaf deployment contains of creating a bootable SD card with a first-stage bootloader (Tow-Boot) and creating USB media with the Ghaf image:
```

Comment:
This is the old sentence, but could you please update it as well by changing:

contains of creating a bootable SD card with a first-stage bootloader (Tow-Boot) and creating USB media with the Ghaf image -> consists of creating a bootable SD card with a first-stage bootloader (Tow-Boot) and USB media with the Ghaf image.

Reply:
Done.